dotcom-rendering/cypress/integration/parallel-6/paid.content.spec.js

```diff
@@ -0,0 +1,65 @@
+/* eslint-disable mocha/no-setup-in-describe */
+import { setLocalBaseUrl } from '../../lib/setLocalBaseUrl.js';
+import { cmpIframe } from '../../lib/cmpIframe';
+import { privacySettingsIframe } from '../../lib/privacySettingsIframe';
+import { storage } from '@guardian/libs';
+
+// It is important to use this article for this test because its commercialProperties
+// coming from the CAPI object are the same for all editions. This way we are making sure
+// the Branding island the test expects will be in the DOM. If an article with different
+// commercialProperties across editions were to be used, the test would have different
+// outcome when running locally (UK edition) and when running in CI. For example,
+// it could be US edition if the CI server runs in US. The best way to deal with
+// this would be to control the edition in the test whether by setting the GU_EDITION
+// cookie or by selecting the edition in the UI. Unfortunately, this is not possible
+// at this point of the migration.
```

Comment:
I hope this comment makes sense!

Reply:
Just wondering why we can't use the `GU_EDITION` cookie at this point of the migration

Reply:
I was hoping the part of the sentence where this argument becomes illogical would go unnoticed 🙈  but yes you are right 🙂  We tried setting the cookie in the test but without success and this has nothing to do with the current point of the migration. Happy to give it another go though after we merge this!

Reply:
Updated the comment to make it clear!